packages/sync-repo-settings/package.json

```diff
@@ -29,7 +30,8 @@
     "ajv": "^8.0.5",
     "extend": "^3.0.2",
     "gcf-utils": "^7.1.3",
-    "js-yaml": "^4.0.0"
+    "js-yaml": "^4.0.0",
+    "yargs": "^17.0.0-candidate.6"
```

Comment:
Why are we using a `-candidate.6`?  Is this where the `latest` tag is on yargs?

Reply:
The feature I was using in the args parser for having a param be "required" either from the command line or an environment variable. Marking it as required, would force you to set it from the CLI, regardless of any default value you provide (maybe this is a bug in yargs).